src/components/stac/collection.tsx

```diff
@@ -323,9 +307,7 @@ function CollectionButtons({ collection, ...rest }: CollectionButtonProps) {
             <Dialog.Content>
               <Dialog.Header></Dialog.Header>
               <Dialog.Body>
-                <LayersProvider setLayers={undefined}>
-                  <Value value={collection}></Value>
-                </LayersProvider>
+                <Collection collection={collection} map={false}></Collection>
```

Comment:
I don't _love_ using a prop to control whether the component renders to the map in this way, but I'm haven't thought of a better way yet.